.github/workflows/publish.js.yml

```diff
@@ -0,0 +1,40 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node and then publish the new bundle to NPM
+
+name: Publish CD
+
+
+on:
+  workflow_dispatch:
+
+jobs:
+  quality:
+    runs-on: ubuntu-latest
+    strategy:
```

Comment:
Can we be consistent and add this code also in the "automatic" job (i.e. `.github/workflows/quality.js.yml`) and below?